test/assertions/like-pattern-concordance.js

```diff
@@ -0,0 +1,21 @@
+const test = require('@ava/test');
+
+const likePatternConcordance = require('../../lib/like-pattern-concordance');
```

Comment:
It should be possible to write an integration test, instead of requiring the module directly.

Reply:
There is an integration test, not in `ava` though.

Reply:
Yea we can remove it.